azkaban-common/src/main/java/azkaban/imagemgmt/daos/RampRuleDao.java

```diff
@@ -0,0 +1,23 @@
+package azkaban.imagemgmt.daos;
+
+import azkaban.imagemgmt.models.ImageRampRule;
+
+
+/**
+* Data access object (DAO) for accessing image ramp rule that deny an image ramping version for flows.
+* This interface defines add/remove/modify an image ramp rule.
+* */
+public interface RampRuleDao {
+
+  boolean isHPRule(String ruleId);
+
+  /*
+  * Insert a new row into DB for ramp rule
+  * */
+  int addRampRule(ImageRampRule rule);
```

Comment:
Please use keyword final wherever applicable. Please follow the coding guidelines of Azkaban.